core/types/block.go

```diff
@@ -188,6 +188,42 @@ func (h *Header) EmptyReceipts() bool {
 	return h.ReceiptHash == EmptyReceiptsHash
 }
 
+func (h *Header) ValidateBlockNumberOptions4337(minBlockNumber *big.Int, maxBlockNumber *big.Int) error {
```

Comment:
Can you add a comment/documentation for this and next methods too?

Reply:
[Done](https://github.com/maticnetwork/bor/pull/945/commits/abe87af0bb99b7995b621af5eb41d50b4a964e0f).